website/content/docs/agent/config-entries/service-intentions.mdx

```diff
@@ -70,6 +75,44 @@ Sources = [
 ]
 ```
 
+</Tab>
+<Tab heading="Kubernetes YAML">
+
+Grant some clients more REST access than others:
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ServiceIntentions
+metadata:
+  name: api
+spec:
+  destination:
+    name: api
+  sources:
+    - name: admin-dashboard
+      permissions:
+        - action: allow
+          http:
+            pathPrefix: /v2
+            methods: ['GET', 'PUT', 'POST', 'DELETE', 'HEAD']
```

Comment:
I go back and forth here about `[]` for slices vs `- entry` but for methods using the `[]` format seems cleaner.